extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -220,6 +222,8 @@ private Project createProject(ApplicationInfoBuildItem app, ArchiveRootBuildItem
         Project project = FileProjectFactory.create(archiveRootBuildItem.getArchiveLocation().toFile());
         BuildInfo buildInfo = new BuildInfo(app.getName(), app.getVersion(),
                 "jar", project.getBuildInfo().getBuildTool(),
+                archiveRootBuildItem.getArchiveRoot()
+                        .resolve(String.format(OUTPUT_ARTIFACT_FORMAT, app.getName(), app.getVersion())),
```

Comment:
That won't work. You can configure the name and the suffix in `PackageConfig`.

See https://github.com/quarkusio/quarkus/blob/master/core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java#L179 for instance.

Reply:
Aha, nice catch! I didn't even remember that is was configurable

Reply:
I think it's fixed now. OK @gsmet?